packages/bot/src/stores/flyout-store.js

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable no-underscore-dangle */
+import React                  from 'react';
+import { observable, action } from 'mobx';
+import FlyoutBlock            from '../components/flyout-block.jsx';
+
+export default class FlyoutStore {
+    block_listeners            = [];
+    block_workspaces           = [];
+    flyout_min_width           = 400;
```

Comment:
@aaron-binary try not to use inline style. why not use css?

Reply:
The width of the flyout is dynamic because block dimensions can change through translations and user-generated blocks. We could set the min-width in CSS, but we will still have to inline the width when the current category exceeds this width, so might as well keep it here instead.